### PR TITLE
feat(wmg): Getting Started

### PR DESCRIPTION
**Pull request by tihuan**
### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
Screen Shot 2022-03-02 at 12 46 55 PM

Screen Shot 2022-03-02 at 12 42 56 PM

demo

## Known Issues
1. Still need the real link for Step 3's "Learn More". Which is covered in https://app.zenhub.com/workspace/o/chanzuckerberg/single-cell-data-portal/issues/1854

### CODE REVIEW

File: frontend/src/views/WheresMyGene/components/Beta/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <StyledCallout intent="info" dismissed={false}>
+      This feature is in beta. If you have any suggestions or feedback, please{" "}
+      <SubmitIssue
+        href="https://github.com/chanzuckerberg/single-cell-data-portal/issues/new/choose"
+        target="_blank"
+        rel="noopener"
+      >
+        submit an issue on our GitHub
+      </SubmitIssue>
```
Comment (seve) [inline frontend/src/views/WheresMyGene/components/Beta/index.tsx:13]: Non-blocker, but I don't think GitHub issues are the right avenue for suggestions or feedback

**Review by seve — APPROVED** [review]

File: frontend/src/views/WheresMyGene/components/Beta/index.tsx (hunk 1 above)
Comment (tihuan) [reply frontend/src/views/WheresMyGene/components/Beta/index.tsx:13]: ah good call out! What's the alternative you have in mind?

CC: @signechambers1 

Thank you!

Comment (seve) [reply frontend/src/views/WheresMyGene/components/Beta/index.tsx:13]: Discussions!

Comment (seve) [reply frontend/src/views/WheresMyGene/components/Beta/index.tsx:13]: Gh Discussions!

**Pull request merged**